platform-common-testing/src/test/java/com/softicar/platform/common/testing/AssertsTest.java

```diff
@@ -9,7 +9,7 @@
 	@Test
 	public void testAssertThrows() {
 
-		Asserts.assertThrows(NumberFormatException.class, () -> Integer.parseInt("foo"));
+		Asserts.assertException(NumberFormatException.class, () -> Integer.parseInt("foo"));
```

Comment:
Because the method under test calls `assertException()` now, the test method must be renamed accordingly.

Reply:
Nice catch. I did not even realize that we have a test for this.